test/main.cpp

```diff
@@ -0,0 +1,4 @@
+#include "ostream"
```

Comment:
wrong way to include

Reply:
https://github.com/R2D2-2019/flame_sensor/pull/1/commits/918f1b28e4fa2fe36c2e2e5b2060e1d946abe208